src/repos/tree/nodes.ts

```diff
@@ -21,17 +21,15 @@ export class RepositoryNode extends TreeItem {
       this.description = "Primary";
     }
 
-    if (repo.branch !== Repository.DEFAULT_BRANCH) {
-      this.contextValue += ".branch";
-      this.description = repo.branch;
-    }
+    this.contextValue += ".branch";
```

Comment:
Makes sense, I am reverting this change for now: "main" will still be showed as branch name since `Repository.DEFAULT_BRANCH` will still point to `master`

Reply:
Sounds good!